src/peft/utils/save_and_load.py

```diff
@@ -134,6 +135,10 @@ def renamed_dora_weights(k):
         else:
             raise NotImplementedError
 
+    elif config.peft_type == PeftType.CUSTOM_TOKENS:
+        # TODO maybe add "{adapter_name}." as prefix to the filter string?
+        to_return = {k: state_dict[k] for k in state_dict if "custom_tokens_delta_tokens" in k}
```

Comment:
Is this block needed or would not already be covered by the `elif config.peft_type in list(PeftType):` below?

Reply:
Ah, of course it is :)